### PR TITLE
chore: add workflow to run go test

### PR DESCRIPTION
**Pull request by rubiojr**
Builds timelinize and runs existing tests.

Adds a small typo fix for a test broken in recent, linter related, refactor.

### CODE REVIEW

File: datasources/media/metadata_test.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func TestSplitCamelCaseIntoWords(t *testing.T) {
 		"GPSAltitudeRef":                   "GPS Altitude Ref",
 		"GPSAltitude":                      "GPS Altitude",
 		"GPSTimeStamp":                     "GPS Time Stamp",
-		"GPSSatelites":                     "GPS Satellites", // OK so it's misspelled in the EXIF data I encountered!
+		"GPSSatelites":                     "GPS Satelites", // OK so it's misspelled in the EXIF data I encountered!
```
Comment (mholt) [inline datasources/media/metadata_test.go:121]: I actually want it spelled correctly in Timelinize though :upside_down_face:

**Review by mholt — CHANGES_REQUESTED** [review]
Thanks for this!

The typo thing is intentional though :+1:

File: datasources/media/metadata_test.go (hunk 1 above)
Comment (rubiojr) [reply datasources/media/metadata_test.go:121]: Ahhh I see 👍 , I'll revert that.

Comment (rubiojr) [reply datasources/media/metadata_test.go]: @mholt I reverted the typo fix in https://github.com/timelinize/timelinize/pull/40/commits/4da830834f4db50858b4322db819efcfc19c4253, but that'll make the test `TestSplitCamelCaseIntoWords` fail. I've been trying to figure out a way to fix it, but that'd require also correct the spelling somehow in `splitCamelCaseIntoWords(s string)`, in addition to to the word splitting.

How do you see that working?

Comment (rubiojr) [reply datasources/media/metadata_test.go]: On a second read, I'm not entirely sure I understand your intentions 🙃

Comment (mholt) [reply datasources/media/metadata_test.go]: OH wow it's a test file. I didn't even notice that :laughing: I was just going through lint errors really fast.

We can just remove this line. That's totally my bad!

Comment (rubiojr) [reply datasources/media/metadata_test.go]: Done! :wrench:

**Review by mholt — APPROVED** [review]
Thanks!!

**Pull request merged**